### PR TITLE
go-packr: init at 1.13.7

### PR DESCRIPTION
**Pull request by edude03**
###### Motivation for this change

Breaking up  #53691 into nice commits / PRs. 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by tomfitzhenry** [comment]
packr has been updated to v2. Should that be packaged instead?

https://github.com/gobuffalo/packr/blob/master/v2/README.md

File: pkgs/development/tools/go-packr/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  goDeps = ./deps.nix;
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/gobuffalo/packr";
+    sha256 = "0hs62w1bv96zzfqqmnq18w71v0kmh4qrqpkf2y8qngvwgan761gd";
+  };
+
+  meta = with stdenv.lib; {
+    license = licenses.mit;
```
Comment (ryantm) [inline pkgs/development/tools/go-packr/default.nix]: Please add `meta.description` and `meta.homepage`.

File: pkgs/development/tools/go-packr/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "go-packr-${version}";
+  version = "v1.13.7";
```
Comment (ryantm) [inline pkgs/development/tools/go-packr/default.nix]: Please update to the latest stable version currently "2.7.1".
```suggestion
  version = "2.7.1";
```

File: pkgs/development/tools/go-packr/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  rev = "5a2cbb54c4e7d482e3f518c56f1f86f133d5204f";
+
+  goPackagePath = "github.com/gobuffalo/packr";
+
+  goDeps = ./deps.nix;
+
+  src = fetchgit {
+    inherit rev;
```
Comment (ryantm) [inline pkgs/development/tools/go-packr/default.nix]:
```suggestion
    rev = "v${version}";
```

File: pkgs/development/tools/go-packr/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "go-packr-${version}";
+  version = "v1.13.7";
+  rev = "5a2cbb54c4e7d482e3f518c56f1f86f133d5204f";
```
Comment (ryantm) [inline pkgs/development/tools/go-packr/default.nix]:
```suggestion
```

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by edude03** [comment]
Hey @ryantm thanks for reviewing the PR, I appreciate the feedback :)

Unfortunately my company has deprecated concourse internally and thus this PR has gone unmaintained. 

How would you suggest we proceed? Should we fix these last issues and merge it, or do you think we should close it until someone else finds a need for it?

**Comment by ryantm** [comment]
@edude03 Personally, I think of myself as a radical inclusionist and would want everything packaged if possible, but, I think most people expect packaged things to have a maintainer, so it is probably best to close it until someone else who wants to maintain it adds it back.

**Comment by edude03** [comment]
Closing because unmaintained

**Pull request closed without merging**